### DISCUSSION TITLE
Do I still need to lock conversation for Announcements type posts?

### DISCUSSION THREAD

**Discussion by laukaichung** [giscus/giscus#959] [Q&A] [answered]
The document says it is recommended to use a category with the Announcements type so that new discussions can only be created by maintainers and giscus. But it seems that a visitor is still able to post a comment for the "Announcements" category. Do I still need to go to the discussion thread and enable "lock conversation" to prevent visitors from posting new comments?

**Comment by laymonage** [comment, marked as answer]
Yes, the announcements type only prevents the creation of discussions, not comments. You need to lock the discussion to prevent new comments.
